lib/utils/cli.go

```diff
@@ -49,65 +52,91 @@ const (
 )
 
 // InitLogger configures the global logger for a given purpose / verbosity level
-func InitLogger(purpose LoggingPurpose, level logrus.Level) {
+func InitLogger(purpose LoggingPurpose, level slog.Level) {
 	logrus.StandardLogger().ReplaceHooks(make(logrus.LevelHooks))
-	logrus.SetLevel(level)
+	logrus.SetLevel(logutils.SlogLevelToLogrusLevel(level))
 	switch purpose {
 	case LoggingForCLI:
 		// If debug logging was asked for on the CLI, then write logs to stderr.
 		// Otherwise, discard all logs.
-		if level == logrus.DebugLevel {
-			debugFormatter := logutils.NewDefaultTextFormatter(trace.IsTerminal(os.Stderr))
+		if level == slog.LevelDebug {
+			enableColors := trace.IsTerminal(os.Stderr)
+			debugFormatter := logutils.NewDefaultTextFormatter(enableColors)
 			_ = debugFormatter.CheckAndSetDefaults()
```

Comment:
Not your change but maybe worth a comment as to why we don't need to check for an error here.